RussianFilter/sections/adservers.txt

```diff
@@ -5,6 +5,10 @@
 ! `window.adswellBaseDomain`
 ! https://damskiy.xyz/vendor/lib/oY.js
 !
+||xokikdassjjhrn.com^$document,popup
+||tmnlsmvgrkoj.com^$document,popup
+||glygrebedu.pro^
+||tmnlsmvgrkoj.com^
```

Comment:
I guess those ones are adservers. Why you didn't use `$third-party`?
Also missed `||autoizers.info^$third-party`.

Reply:
Ok, I agree.